site/docs/tools/index.md

```diff
@@ -25,7 +25,7 @@ the following:
 |Create Branch| :material-check: | :material-check: | :material-check:|
 |Create Tag| :material-check: | :material-check: |:material-check:|
 |Iceberg Tables|:material-check:|:material-check:| |
-|Delta Lake Tables|:material-check:|:material-check:| |
+|Delta Lake Tables||:material-check:| |
```

Comment:
should we also remove `Spark2` from the feature matrix here?

Reply:
If I am not mistaken, Spark2 is still relevant to Iceberg and thus, just removed the check mark from the Delta Lake / Spark2 matrix.